### ISSUE TITLE
bye

### ISSUE THREAD

**Issue by elfeliza** [BerezinDaniil/ML_OPS#1]

**Comment by BerezinDaniil** [comment]
Не пиши мне больше, пожалуйста

**Comment by elfeliza** [comment]
https://www.kaggle.com/datasets/shrutimehta/nasa-asteroids-classification

**Comment by BerezinDaniil** [comment]
class my_model(nn.Module):
  def __init__(self, features, num_classes):
        super(my_model, self).__init__()
        self.linear_1 = nn.Linear(features, 256)
        self.linear_2 = nn.Linear(256, num_classes)
        #self.linear_3 = nn.Linear(256, 64)
        #self.linear_4 = nn.Linear(128, 64)
        #self.linear_5 = nn.Linear(64, num_classes)
        self.act_1 = nn.ReLU()
        #self.act_2 = nn.ReLU()
        #self.act_3 = nn.ReLU()
        #self.act_4 = nn.ReLU()
        #self.act_5 = nn.ReLU()
        self.batchnorm_1 = nn.BatchNorm1d(features)
        #self.batchnorm_2 = nn.BatchNorm1d(256)
        #self.batchnorm_3 = nn.BatchNorm1d(128)
        #self.batchnorm_4 = nn.BatchNorm1d(64)

  def forward(self, x):
        x = self.batchnorm_1(x)
        x = self.linear_1(x)
        x = self.act_1(x)
        x = self.linear_2(x)
        return F.softmax(x, dim=1)


device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = my_model(NUM_FEATURES, NUM_CLASSES).to(device)

**Comment by BerezinDaniil** [comment]
[dataset_nasa.csv](https://github.com/BerezinDaniil/ML_OPS/files/13478458/dataset_nasa.csv)

**Comment by elfeliza** [comment]
https://www.kaggle.com/datasets/abhishek14398/heart-disease-classification

**Comment by elfeliza** [comment]
https://www.kaggle.com/datasets/alexteboul/diabetes-health-indicators-dataset

**Comment by elfeliza** [comment]
https://www.kaggle.com/datasets/shrutimehta/nasa-asteroids-classification

**Comment by elfeliza** [comment]
[dataset_nasa.csv](https://github.com/BerezinDaniil/ML_OPS/files/13517336/dataset_nasa.csv)

**Comment by BerezinDaniil** [comment]
Блять ты кому звонишь сынок ебаный 
[Uploading train_NASA.csv…]()

**Comment by BerezinDaniil** [comment]
[train_NASA.csv](https://github.com/BerezinDaniil/ML_OPS/files/13517356/train_NASA.csv)
[test_NASA.csv](https://github.com/BerezinDaniil/ML_OPS/files/13517358/test_NASA.csv)

**Comment by BerezinDaniil** [comment]
удаляй ветку 
к хуям
